server/gql/models/tests/studentSubjects/customUpdateResolver.test.js

```diff
@@ -0,0 +1,74 @@
+import { studentsTable, subjectsTable } from '@server/utils/testUtils/mockData';
+import { getResponse, mockDBClient, resetAndMockDB } from '@server/utils/testUtils';
+import { get } from 'lodash';
+
+describe('StudentSubject graphql-server-db customUpdateResolver tests', () => {
+  let dbClient;
+  beforeEach(() => {
+    dbClient = mockDBClient();
+    resetAndMockDB(null, {}, dbClient);
+  });
+  const id = 1;
+  const updateStudentSubjectMutation = `
+  mutation {
+    updateStudentSubject (
+      id: ${id},
+      studentId: ${studentsTable[0].id},
+      subjectId: ${subjectsTable[0].id}
+    ) {
+      id
+      studentId
+      subjectId
+    }
+  }
+  `;
+  it('should have mutation to update a new student', async () => {
+    jest.spyOn(dbClient.models.studentSubjects, 'update');
+    const response = await getResponse(updateStudentSubjectMutation);
+    const result = get(response, 'body.data.updateStudentSubject');
+    expect(result).toBeTruthy();
```

Comment:
let's check against the shape of the result, not just for truthy

Reply:
for the first time, it is checking truthy, after that checking with the shape of the result.